### PR TITLE
Updated instructions for trusting the certificate.

### PR DESCRIPTION
**Pull request by MIchaelMainer**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -130,4 +86,42 @@ Specifying `--js` tells the generator to use JavaScript.
...
+  		  
+Launch the local HTTPS site on `https://localhost:3000` by simply typing the following command in your console:
+
+```bash
+$ npm start
+```
+
+Browsersync will start an HTTPS server, which includes a self-signed SSL cert that your development environment must trust. Refer to our doc, [Adding Self-Signed Certificates as Trusted Root Certificate](src/docs/ssl.md), for instructions on how to do this.
+
+Browse to the 'External' IP address listed in your console to test your web app across multiple browsers and devices that are connected on your local network.
```
Comment (kbrandl) [inline readme.md]: Seems like the "Adding Self-Signed Certificates as Trusted Root Certificate" page contains guidance for the same task that you've enumerated above (in steps 1-6) -- except that the guidance/instructions in that article is different than steps 1-6 above.   Perhaps consider updating the "Adding Self-Signed Certificates as Trusted Root Certificate" page with correct/complete guidance for all platforms, continue linking to that page from this README, and remove steps 1-6 above (i.e., the **Important** alert) from this article?

File: readme.md [hunk 2]
```diff
@@ -130,4 +86,42 @@ Specifying `--js` tells the generator to use JavaScript.
 
 >**Note:** Do not use this flag when you pass `react` as framework argument.
 
+## Running the Generated Site
+
+Office Add-ins must be hosted in an HTTPS site. Yo Office generates a [Browsersync](https://browsersync.io/) configuration file (`bsconfig.json`) that references a self-signed certificate. Your computer will need to trust the certificate before you can use the generated add-in. 
+
+> **Important:** Before you continue with this Readme, take the following steps (Windows):
```
Comment (kbrandl) [inline readme.md]: (See note below. Consider removing this note if you update the "Adding Self-Signed Certificates as Trusted Root Certificate" page with correct/complete guidance.)

File: readme.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Like other Yeoman generators, this simply creates the scaffolding of files for y
 - Project
 - Word
 
-Choose to create the Office projects using plain HTML, CSS & JavaScript (*mirroring the same projects that Visual Studio creates*) or create Angular-based projects.
+Choose to create Office Add-in projects using plain HTML, CSS & JavaScript (*mirroring the same projects that Visual Studio creates*) or create Angular-based projects.
```
Comment (kbrandl) [inline readme.md]: Should this also include react-based projects?

Choose to create Office Add-in projects using plain HTML, CSS & JavaScript (*mirroring the same projects that Visual Studio creates*), create Angular-based projects, or create React-based projects.

Comment (MIchaelMainer) [reply readme.md]: I don't know. I have no idea about the state of the React templates. I just wonder if it was left out on purpose.

File: readme.md (hunk 1 above)
Comment (MIchaelMainer) [reply readme.md]: I suspect you can do this a couple of different ways. I'm inclined to leave the the steps in the readme and point to the "Adding Self-signed..." page as the instructions for Mac.

**Pull request merged**